### PR TITLE
add job preferences

### PR DESCRIPTION
**Pull request by manukao**
have a look please, thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,7 +57,6 @@ <h3>Bonn</h3>
 
       <div id="Latest">
         <h3>Latest</h3>
-
```
Comment (DominicWebDev) [inline index.html:60]: to much space

**Review by DominicWebDev — APPROVED** [review]
good work as always

**Review by Satttoshi — APPROVED** [review]
some of div elements could be replaced, anyways you nailed it!
LGTM 🚀

**Review by FrankSiyli — APPROVED** [review]
you wanna be my DIV-Buddy?😄

 LGTM👌

**Comment by toshydev** [comment]
Wow! Nice work!  👌

File: index.html (hunk 1 above)
Comment (manukao) [reply index.html:60]: fixed, thank you

**Pull request merged**